### PR TITLE
Implement setters in the openzwave adapter

### PR DESCRIPTION
**Pull request by julienw**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -197,21 +197,59 @@ impl Watchers {
...
 }
 
-fn to_open_closed(value: &ValueID) -> Option<Value> {
-    debug_assert_eq!(value.get_type(), ValueType::ValueType_Bool);
+fn ozw_vid_as_tax_value(vid: &ValueID) -> Option<Value> {
+    let cc = vid.get_command_class();
+    if cc.is_none() {
+        return None;
+    }
+    let cc = cc.unwrap();
```
Comment (fabricedesre) [inline src/lib.rs]: cc is unused ?

File: src/lib.rs [hunk 2]
```diff
@@ -197,21 +197,59 @@ impl Watchers {
...
-        Value::OpenClosed(
-            if val { OpenClosed::Open } else { OpenClosed::Closed }
-        )
-    })
+fn set_ozw_vid_from_tax_value(vid: &ValueID, value: Value) -> Result<(), TaxError> {
+    let cc = vid.get_command_class();
+    if cc.is_none() {
+        return Err(TaxError::InternalError(InternalError::GenericError(format!("Unknown command class: {}", vid.get_command_class_id()))));
+    }
+    let cc = cc.unwrap();
```
Comment (fabricedesre) [inline src/lib.rs]: here too?

**Comment by fabricedesre** [comment]
What does "tax" mean in this code?

**Comment by julienw** [comment]
> What does "tax" mean in this code?

This usually a shortcut for "taxonomy". This is not so good though. Maybe the longer "taxo" could be better as a shortcut.

**Pull request merged**